Dockerfile

```diff
@@ -132,9 +132,11 @@ ENV PATH /opt/lcov-1.14/bin:$PATH
 # Install rust compiler
 ENV PATH /opt/cargo/bin:$PATH
 ENV RUSTUP_HOME=/opt/rustup
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | CARGO_HOME=/opt/cargo sh -s -- --default-toolchain 1.42.0 -y
+# Also update ./rust-toolchain when changing the default toolchain.
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | CARGO_HOME=/opt/cargo sh -s -- --default-toolchain nightly-2020-03-25 -y
```

Comment:
The rust-toolchain is very helpful! Could we do something like this?

```
COPY rust-toolchain /tmp/
RUN ... | CARGO_HOME=/opt/cargo sh -s -- --default-toolchain $(cat /tmp/rust-toolchain) -y
```

So to attempt to keep them in sync.

Reply:
Can be fragile, e.g. if someone slips in a newline in there, it will break the docker build without anyone noticing.

Reply:
Sure. Someone could also update Dockerfile while forgetting to update rust-toolchain or the other way around. :)
Either way it fine with me!